File: Data Structures & Algorithms/two-integer-sum/submission-6.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hset = {}
        
        for i, v in enumerate(nums):
            diff = target - v
            if diff in hset:
                return [hset[diff],i]
            else:
                hset[v] = i

File: Data Structures & Algorithms/power-of-two/submission-6.java
class Solution {
    public boolean isPowerOfTwo(int n) {
        if(n<=0)
        {
            return false;
        }
        int x = n;
        while(x%2==0)
        {
                x=x/2;
        }
        if(x==1)
        {
            return true;
        }
        else{
            return false;
        }

    }
}